Gemfile

```diff
@@ -25,6 +25,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 
+gem 'puma'
+
 group :development, :test do
```

Comment:
Concerning gems in development AND test groups:
- https://github.com/thoughtbot/factory_bot_rails#configuration
- https://github.com/rspec/rspec-rails#installation

Reply:
Done.